dali/operators/generic/reshape.cc

```diff
@@ -247,10 +266,18 @@ void Reshape<Backend>::CalculateOutputShape(const Workspace &ws) {
   switch (shape_source_) {
     case ShapeSource::Arg:
       if (use_rel_shape_) {
+        if (!src_dims_.empty()) {
```

Comment:
This fails to capture the case of dropping _all_ dimesnions from a scalar value.